lang/el.json

```diff
@@ -108,17 +108,17 @@
 	"ErrorUnknown": "Η ενέργεια δεν ήταν δυνατόν να εκτελεστεί. (Σφάλμα %1)",
 	"Errors": {
 		"10": "Λανθασμένη Εντολή.",
-		"11": "Το resource type δεν ήταν δυνατόν να προσδιοριστεί.",
-		"12": "Το resource type δεν είναι έγκυρο.",
-		"13": "The connector configuration file is invalid.",
-		"14": "Invalid connector plugin: %1.",
+		"11": "Ο τύπος πόρου(resource type) δεν ήταν δυνατόν να προσδιοριστεί.",
+		"12": "Ο τύπος πόρου(resource type) δεν είναι έγκυρος.",
+		"13": "Το αρχείο ρυθμίσεων του πρόσθετου(plugin) σύνδεσης είναι άκυρο.",
+		"14": "Άκυρο πρόσθετο(plugin) σύνδεσης: %1.",
```

Comment:
Hi Aris,

Answer 1: connector is not a plugin. CKFinder consists of 2 parts: client-side part written in JavaScript and server-side part (also known as the "connector", currently in PHP). CKFinder plugins come in two types: the client-side plugins and the server-side plugins. This message is displayed when one of the server-side ("connector") plugins is incorrect.

Hope this helps!